openhtf/plugs/usb/__init__.py

```diff
@@ -52,9 +72,21 @@ def _open_usb_handle(**kwargs):
   Returns:
     Instance of UsbHandle.
   """
+
+  serial = None
   if conf.Config().usb_server:
     return None
   else:
+    remote_usb = conf.Config().remote_usb
+    if remote_usb:
+      if remote_usb.has_key('Ethersync'):
```

Comment:
I was thinking if remote_usb is set to 'ethersync', we require another top-level key called 'ethersync' that holds the ethersync configuration info. Definitely open to discussion on this, though. At the same time, we should think about standardizing our config keys on either CamelCase or underscore_notation.